src/tracers.jl

```diff
@@ -211,7 +211,7 @@ gradient(d::Dual{P,T}) where {P,T<:GradientTracer} = gradient(d.tracer)
 gradient(d::Dual{P,T}) where {P,T<:HessianTracer} = gradient(d.tracer)
 hessian(d::Dual{P,T}) where {P,T<:HessianTracer} = hessian(d.tracer)
 
-function Dual{P,T}(x::Number) where {P<:Number,T<:AbstractTracer}
+function Dual{P,T}(x::Real) where {P<:Real,T<:AbstractTracer}
     return Dual(convert(P, x), empty(T))
 end
```

Comment:
Needs support for returning a `Complex{T}` when called with `Complex` number.

Reply:
#102